server/bleep/src/indexes/file.rs

```diff
@@ -285,20 +279,26 @@ impl Indexer<File> {
         // XXX: can we use the bloop query language here instead?
         let query_parser = QueryParser::for_index(
             file_index,
-            vec![self.source.repo_disk_path, self.source.relative_path],
+            vec![self.source.repo_ref, self.source.relative_path],
```

Comment:
Why this change?

Reply:
Moves our logic from disk-based to a purely symbolic space, consistent with removing IDs that carry semantic meaning.